### PR TITLE
TECH-1879_honeybadger-exception-context - if an exception_context is …

### PR DESCRIPTION
**Pull request by codyjca**
If an exception_context is provided, include it when posting notifications to honey badger.

@nburwell here is the update to the exception handling gem. This should cover the VBR case we were looking at.

image

### CODE REVIEW

File: lib/exception_handling/exception_info.rb [hunk 1]
```diff
@@ -249,6 +249,7 @@ def dump_hash( h, indent_level = 0 )
     def enhanced_data_to_honeybadger_context
       data = enhanced_data.dup
       data[:server] = ExceptionHandling.server_name
+      data[:context] = @exception_context if @exception_context.present?
```
Comment (nburwell) [inline lib/exception_handling/exception_info.rb]: are we going to regret it being "context > context" in honeybadger?  I know that's what we had said before, just my only concern being confusion around the two layers there. Maybe "extra_context" or "invoca_context" ?

Comment (codyjca) [inline lib/exception_handling/exception_info.rb]: Probably, it is a little weird.  In the web side we call this param exception_context.

Comment (codyjca) [inline lib/exception_handling/exception_info.rb]: Renamed to "exception_context"

**Comment by codyjca** [comment]
Ready for another review.

**Comment by nburwell** [comment]
👍

**Pull request merged**